oop_verify.py

```diff
@@ -124,14 +124,9 @@ class ProjectFilesVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
-        self.minimum_size = 410
+        self.minimum_size = 380
         self.page_elements = [
-            '#nodeTitleEditable',                                # Title
```

Comment:
@sf2ne All those elements were copy-pasted along with the project dashboard subclass to lay out all the other subclasses. They don't appear on other pages, except registration dashboard.